### PR TITLE
fix: Remove deprecated object-form attrs({}) keys

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: **Bugfix**

### Description

`object-form attrs({})` keys are now deprectaed in `styled-components`. This PR switches to the new `attr(props => ({}))` syntax which will remove the warning (see screenshot).

simulator_screenshot_D1B6E35F-CECE-4C27-8E4B-8D9756EA16AB

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Remove deprecated object-form attrs({}) keys in styled-components - ole

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Remove deprecated object-form attrs({}) keys in styled-components - ole)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e8b191934aa5c8a2c2f631bcabbc0b9d6d03d37c
</p>

File: src/lib/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tsx [hunk 1]
```diff
@@ -189,7 +189,7 @@ export const OnboardingCreateAccountScreenWrapper: React.FC<OnboardingCreateAcco
             </>
           )}
         </Box>
-        <Spacer mt={50} />
+        {/* <Spacer mt={50} /> */}
```
Comment (pvinis) [inline src/lib/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tsx:192]: whats up with this one? accidental? or should we remove it or keep it?

**Review by pvinis — APPROVED** [review]

**Comment by pvinis** [comment]
Just one comment. Once it's resolved, feel free to merge, I assigned you :). Thanks for this PR!

File: src/lib/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tsx (hunk 1 above)
Comment (olerichter00) [reply src/lib/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tsx:192]:
```suggestion
        <Spacer mt={50} />
```

Comment (olerichter00) [reply src/lib/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tsx]: Nice catch! I reverted the change....

**Comment by olerichter00** [comment]
#squash-on-green

**Pull request merged**